### PR TITLE
Add missing options for chef-server-ctl backup and restore

### PR DESCRIPTION
**Pull request by lbakerchef**
Signed-off-by: Lincoln Baker <lbaker@chef.io>

### CODE REVIEW

File: themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md [hunk 1]
```diff
@@ -3,4 +3,20 @@ This subcommand has the following options:
 `-y`, `--yes`
 
 :   Use to specify if the Chef Infra Server can go offline during
-    tar.gz-based backups.
\ No newline at end of file
+    tar.gz-based backups.
+
+`--pg-options`
+
+:   Specify additional options to pass to PostgreSQL during backups.
```
Comment (PrajaktaPurohit) [inline themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md]: could we include some examples of the options?

**Review by PrajaktaPurohit — APPROVED** [review]

File: themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md (hunk 1 above)
Comment (lbakerchef) [reply themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md]: We will skip this for now, as discussed in standup.

**Review by PrajaktaPurohit — APPROVED** [review]

File: themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md [hunk 2]
```diff
@@ -3,4 +3,20 @@ This subcommand has the following options:
 `-y`, `--yes`
 
 :   Use to specify if the Chef Infra Server can go offline during
-    tar.gz-based backups.
\ No newline at end of file
+    tar.gz-based backups.
+
+`--pg-options`
```
Comment (kagarmoe) [inline themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md:8]: Where would a user find the pg options?

File: themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md [hunk 3]
```diff
@@ -3,4 +3,20 @@ This subcommand has the following options:
...
+
+:   Specify additional options to pass to PostgreSQL during backups.
+
+`-c`, `--config-only`
+
+:   Only backup configuration, not data.
```
Comment (kagarmoe) [inline themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md]:
```suggestion
:   Backup the Chef Infra Server configuration **without** backing up data.
```

File: themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md [hunk 4]
```diff
@@ -3,4 +3,20 @@ This subcommand has the following options:
...
+
+`-t`, `--timeout`
+
+:   Set the Maximum amount of time to wait for shell commands.
```
Comment (kagarmoe) [inline themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md]: Does this configuration have a default? I assume that it is set in MS?

**Comment by kagarmoe** [comment]
A few questions

**Review by kagarmoe — CHANGES_REQUESTED** [review]
I have a few questions

File: themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md (hunk 2 above)
Comment (lbakerchef) [reply themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md:8]: https://www.postgresql.org/docs/

File: themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md (hunk 4 above)
Comment (lbakerchef) [reply themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md]: I don't know if it has a default that is set anywhere.  I do know that customers have to set this from time to time to resolve issues.

Comment (lbakerchef) [reply themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md]: This now reads:
```
 Set the maximum amount of time in seconds to wait for shell commands (default 600). This option should be set to greater than 600 for backups taking longer than 10 minutes.
```

File: themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md (hunk 1 above)
Comment (kagarmoe) [reply themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md]:
```suggestion
:   Use to specify additional options to pass to PostgreSQL during backups. See the [PostgreSQL documentation](https://www.postgresql.org/docs/) for more information.
```

File: themes/docs-new/layouts/shortcodes/ctl_chef_server_restore_options.md [hunk 5]
```diff
@@ -9,4 +9,16 @@ This subcommand has the following options:
 
 :   Use to specify that the path to an empty directory to be used during
     the restore process. This directory must have enough disk space to
-    expand all data in the backup archive.
\ No newline at end of file
+    expand all data in the backup archive.
+
+`--pg-options`
+
+:   Specify additional options to pass to PostgreSQL during restore.
```
Comment (kagarmoe) [inline themes/docs-new/layouts/shortcodes/ctl_chef_server_restore_options.md]:
```suggestion
:   Use to specify additional options to pass to PostgreSQL during backups. See the [PostgreSQL documentation](https://www.postgresql.org/docs/) for more information.
```

**Review by kagarmoe — CHANGES_REQUESTED** [review]
2 small changes. Ignore the cspell warnings on this PR. We'll fix them elsewhere.

File: themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md (hunk 2 above)
Comment (PrajaktaPurohit) [reply themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md:8]: Looks like https://www.postgresql.org/docs/13/runtime-config.html are the options that the user could set. they would be passed to chef_backup that would set them as PGOPTIONS env variable. https://github.com/chef/chef_backup/blob/main/lib/chef_backup/strategy/backup/tar.rb#L61

Comment (lbakerchef) [reply themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md:8]: @PrajaktaPurohit Thanks!  I added your link.

**Comment by tas50** [comment]
@kagarmoe is this one good with the updates made?

File: themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md [hunk 6]
```diff
@@ -3,4 +3,20 @@ This subcommand has the following options:
 `-y`, `--yes`
 
 :   Use to specify if the Chef Infra Server can go offline during
-    tar.gz-based backups.
\ No newline at end of file
+    tar.gz-based backups.
+
+`--pg-options`
+
+:   Use to specify additional options to pass to PostgreSQL during backups. See the [PostgreSQL documentation](https://www.postgresql.org/docs/13/runtime-config.html) for more information.
```
Comment (kagarmoe) [inline themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md]:
```suggestion
:   Use to specify and pass additional options PostgreSQL during backups. See the [PostgreSQL documentation](https://www.postgresql.org/docs/13/runtime-config.html) for more information.
```

File: themes/docs-new/layouts/shortcodes/ctl_chef_server_restore_options.md [hunk 7]
```diff
@@ -9,4 +9,16 @@ This subcommand has the following options:
 
 :   Use to specify that the path to an empty directory to be used during
     the restore process. This directory must have enough disk space to
-    expand all data in the backup archive.
\ No newline at end of file
+    expand all data in the backup archive.
+
+`--pg-options`
+
+:   Use to specify additional options to pass to PostgreSQL during restore. See the [PostgreSQL documentation](https://www.postgresql.org/docs/13/runtime-config.html) for more information.
```
Comment (kagarmoe) [inline themes/docs-new/layouts/shortcodes/ctl_chef_server_restore_options.md]:
```suggestion
```suggestion
:   Use to specify and pass additional options PostgreSQL during backups. See the [PostgreSQL documentation](https://www.postgresql.org/docs/13/runtime-config.html) for more information.
```
```

File: themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md [hunk 8]
```diff
@@ -3,4 +3,20 @@ This subcommand has the following options:
...
+
+`-c`, `--config-only`
+
+:   Backup the Chef Infra Server configuration **without** backing up data.
+
+`-t`, `--timeout`
+
+:   Set the maximum amount of time in seconds to wait for shell commands (default 600). This option should be set to greater than 600 for backups taking longer than 10 minutes.
```
Comment (kagarmoe) [inline themes/docs-new/layouts/shortcodes/ctl_chef_server_backup_options.md:18]:
```suggestion
:   Set the maximum amount of time in seconds to wait for shell commands. Set to greater than 600 for backups that take longer than 10 minutes. Default: 600.
```

File: themes/docs-new/layouts/shortcodes/ctl_chef_server_restore_options.md [hunk 9]
```diff
@@ -9,4 +9,16 @@ This subcommand has the following options:
...
+
+`-t`, `--timeout`
+
+:   Set the maximum amount of time in seconds to wait for shell commands (default 600). This option should be set to greater than 600 for restores taking longer than 10 minutes.
```
Comment (kagarmoe) [inline themes/docs-new/layouts/shortcodes/ctl_chef_server_restore_options.md]:
```suggestion
:   Set the maximum amount of time in seconds to wait for shell commands. Set to greater than 600 for backups that take longer than 10 minutes. Default: 600.
```

**Review by kagarmoe — CHANGES_REQUESTED** [review]

**Pull request merged**